<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head>
<link rel="icon" href="/favicon.png">
<title>#737778 - python-numpy: insecure use of /tmp (CVE-2014-1858 CVE-2014-1859) - Debian Bug report logs</title>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="/css/bugs.css" type="text/css">

<link rel="canonical" href="&lt;a href=&quot;bugreport.cgi?bug=737778&quot;&gt;737778&lt;/a&gt;">
<script type="text/javascript">
<!--
function toggle_infmessages()
{
        allDivs=document.getElementsByTagName("div");
        for (var i = 0 ; i < allDivs.length ; i++ )
        {
                if (allDivs[i].className == "infmessage")
                {
                        allDivs[i].style.display=(allDivs[i].style.display == 'none' | allDivs[i].style.display == '') ? 'block' : 'none';
                }
        }
}
-->
</script>
</head>
<body>
<h1>Debian Bug report logs - 
<a href="mailto:737778@bugs.debian.org">#737778</a><br>
python-numpy: insecure use of /tmp (CVE-2014-1858 CVE-2014-1859)</h1>
<div class="versiongraph"><a href="version.cgi?info=1;collapse=1;absolute=0;found=python-numpy%2F1%3A1.7.1-3;fixed=python-numpy%2F1%3A1.8.1~rc1-1;package=python-numpy"><img alt="version graph" src="version.cgi?collapse=1;width=2;height=2;package=python-numpy;fixed=python-numpy%2F1%3A1.8.1~rc1-1;found=python-numpy%2F1%3A1.7.1-3;absolute=0"></a></div>
<div class="pkginfo">
  <p>Package:
     <a class="submitter" href="pkgreport.cgi?package=python-numpy">python-numpy</a>;
Maintainer for <a href="pkgreport.cgi?package=python-numpy">python-numpy</a> is <a href="pkgreport.cgi?maint=morph%40debian.org">Sandro Tosi &lt;morph@debian.org&gt;</a>; Source for <a href="pkgreport.cgi?package=python-numpy">python-numpy</a> is <a href="pkgreport.cgi?src=python-numpy">src:python-numpy</a> (<a href="https://tracker.debian.org/pkg/python-numpy">PTS</a>, <a href="https://buildd.debian.org/python-numpy">buildd</a>, <a href="https://qa.debian.org/popcon.php?package=python-numpy">popcon</a>). </p>

</div>

<div class="buginfo">
  <p>Reported by: <a href="pkgreport.cgi?submitter=jwilk%40debian.org">Jakub Wilk &lt;jwilk@debian.org&gt;</a></p>
  <p>Date: Wed,  5 Feb 2014 21:18:02 UTC</p>

<p>Severity: important</p>
<p>Tags: patch, security</p>

<p>Found in version python-numpy/1:1.7.1-3</p>
<p>Fixed in version python-numpy/1:1.8.1~rc1-1</p>

<p><strong>Done:</strong> Julian Taylor &lt;jtaylor.debian@googlemail.com&gt;</p>

<p>Bug is archived. No further changes may be made.<p><p>Forwarded to <a href="https://github.com/numpy/numpy/pull/4262">https://github.com/numpy/numpy/pull/4262</a></p>
</div>

<p><a href="javascript:toggle_infmessages();">Toggle useless messages</a></p><div class="msgreceived"><p>View this report as an <a href="bugreport.cgi?bug=737778;mbox=yes">mbox folder</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;mboxstatus=yes">status mbox</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;mboxmaint=yes">maintainer mbox</a></p></div>

<div class="infmessage"><hr><p>
<a name="1"></a>
<!-- request_addr: debian-bugs-dist@lists.debian.org, jwilk@debian.org, Debian Python Modules Team &lt;python-modules-team@lists.alioth.debian.org&gt; -->
<!-- time:1391635086 -->
<strong>Report forwarded</strong>
to <code>debian-bugs-dist@lists.debian.org, jwilk@debian.org, Debian Python Modules Team &lt;python-modules-team@lists.alioth.debian.org&gt;</code>:<br>
<code>Bug#737778</code>; Package <code>python-numpy</code>.
 (Wed, 05 Feb 2014 21:18:06 GMT) (<a href="bugreport.cgi?bug=737778;msg=2">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=2">mbox</a>, <a href="#1">link</a>).</p></p></div>

<hr><p class="msgreceived"><a name="3"></a><a name="msg3"></a><a href="#3">Message #3</a> received at submit@bugs.debian.org (<a href="bugreport.cgi?bug=737778;msg=3">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=3">mbox</a>, <a href="mailto:737778@bugs.debian.org?body=On%20Wed%2C%205%20Feb%202014%2022%3A16%3A42%20%2B0100%20Jakub%20Wilk%20%3Cjwilk%40debian.org%3E%20wrote%3A%0A%3E%20Package%3A%20python-numpy%0A%3E%20Version%3A%201%3A1.7.1-3%0A%3E%20Severity%3A%20important%0A%3E%20Tags%3A%20security%0A%3E%20%0A%3E%20numpy%2Ff2py%2F__init__.py%20contains%20this%20code%3A%0A%3E%20%0A%3E%20%20%20%20%20%20%20from%20numpy.distutils.exec_command%20import%20exec_command%0A%3E%20%20%20%20%20%20%20import%20tempfile%0A%3E%20%20%20%20%20%20%20if%20source_fn%20is%20None%3A%0A%3E%20%20%20%20%20%20%20%20%20%20%20fname%20%3D%20os.path.join%28tempfile.mktemp%28%29%2B%27.f%27%29%0A%3E%20%20%20%20%20%20%20else%3A%0A%3E%20%20%20%20%20%20%20%20%20%20%20fname%20%3D%20source_fn%0A%3E%20%0A%3E%20%20%20%20%20%20%20f%20%3D%20open%28fname%2C%27w%27%29%0A%3E%20%0A%3E%20From%20the%20tempfile.mktemp%28%29%20docstring%3A%20%E2%80%9CThis%20function%20is%20unsafe%20and%20%0A%3E%20should%20not%20be%20used.%20The%20file%20name%20refers%20to%20a%20file%20that%20did%20not%20exist%20at%20%0A%3E%20some%20point%2C%20but%20by%20the%20time%20you%20get%20around%20to%20creating%20it%2C%20someone%20else%20%0A%3E%20may%20have%20beaten%20you%20to%20the%20punch.%E2%80%9D%0A%3E%20%0A%3E%20%28There%20are%20other%20calls%20to%20tempfile.mktemp%28%29%20in%20the%20numpy%20codebase%2C%20but%20I%20%0A%3E%20haven%27t%20checked%20them.%29%0A%3E%20%0A%3E%20--%20%0A%3E%20Jakub%20Wilk%0A%3E%20%0A%3E%20%0A&amp;In-Reply-To=%3C20140205211642.GA10530%40jwilk.net%3E&amp;References=%3C20140205211642.GA10530%40jwilk.net%3E&amp;subject=Re%3A%20python-numpy%3A%20insecure%20use%20of%20%2Ftmp">reply</a>):</p>
<div class="headers">
<img src="/cgi-bin/libravatar.cgi?email=jwilk%40debian.org" alt="">
<div class="header"><span class="headerfield">From:</span> Jakub Wilk &lt;jwilk@debian.org&gt;</div>
<div class="header"><span class="headerfield">To:</span> Debian Bug Tracking System &lt;submit@bugs.debian.org&gt;</div>
<div class="header"><span class="headerfield">Subject:</span> python-numpy: insecure use of /tmp</div>
<div class="header"><span class="headerfield">Date:</span> Wed, 5 Feb 2014 22:16:42 +0100</div>
</div>
<pre class="message flowed">Package: python-numpy
Version: 1:1.7.1-3
Severity: important
Tags: security

numpy/f2py/__init__.py contains this code:

     from numpy.distutils.exec_command import exec_command
     import tempfile
     if source_fn is None:
         fname = os.path.join(tempfile.mktemp()+&#39;.f&#39;)
     else:
         fname = source_fn

     f = open(fname,&#39;w&#39;)

From the tempfile.mktemp() docstring: “This function is unsafe and 
should not be used. The file name refers to a file that did not exist at 
some point, but by the time you get around to creating it, someone else 
may have beaten you to the punch.”

(There are other calls to tempfile.mktemp() in the numpy codebase, but I 
haven&#39;t checked them.)

-- 
Jakub Wilk


</pre>

<div class="msgreceived"><hr><p>
<a name="4"></a>
<!-- command:forwarded -->
<!-- requester: Julian Taylor &lt;jtaylor.debian@googlemail.com&gt; -->
<!-- request_addr: control@bugs.debian.org -->
<!-- time:1391638870 -->
<!-- new_data:
$new_data = {
              &#39;forwarded&#39; =&gt; &#39;<a href="https://github.com/numpy/numpy/pull/4262">https://github.com/numpy/numpy/pull/4262</a>&#39;
            };
-->
<!-- old_data:
$old_data = {
              &#39;forwarded&#39; =&gt; &#39;&#39;
            };
-->
<strong>Set Bug forwarded-to-address to &#39;<a href="https://github.com/numpy/numpy/pull/4262">https://github.com/numpy/numpy/pull/4262</a>&#39;.</strong>
Request was from <code>Julian Taylor &lt;jtaylor.debian@googlemail.com&gt;</code>
to <code>control@bugs.debian.org</code>.
 (Wed, 05 Feb 2014 22:21:10 GMT) (<a href="bugreport.cgi?bug=737778;msg=5">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=5">mbox</a>, <a href="#4">link</a>).</p></p></div>

<div class="infmessage"><hr><p>
<a name="6"></a>
<!-- request_addr: debian-bugs-dist@lists.debian.org, Debian Python Modules Team &lt;python-modules-team@lists.alioth.debian.org&gt; -->
<!-- time:1391659388 -->
<strong>Information forwarded</strong>
to <code>debian-bugs-dist@lists.debian.org, Debian Python Modules Team &lt;python-modules-team@lists.alioth.debian.org&gt;</code>:<br>
<code>Bug#737778</code>; Package <code>python-numpy</code>.
 (Thu, 06 Feb 2014 04:03:08 GMT) (<a href="bugreport.cgi?bug=737778;msg=7">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=7">mbox</a>, <a href="#6">link</a>).</p></p></div>

<div class="infmessage"><hr><p>
<a name="8"></a>
<!-- request_addr: mmcallis@redhat.com -->
<!-- time:1391659388 -->
<strong>Acknowledgement sent</strong>
to <code>mmcallis@redhat.com</code>:<br>
Extra info received and forwarded to list.  Copy sent to <code>Debian Python Modules Team &lt;python-modules-team@lists.alioth.debian.org&gt;</code>.
 (Thu, 06 Feb 2014 04:03:08 GMT) (<a href="bugreport.cgi?bug=737778;msg=9">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=9">mbox</a>, <a href="#8">link</a>).</p></p></div>

<hr><p class="msgreceived"><a name="10"></a><a name="msg10"></a><a href="#10">Message #10</a> received at 737778@bugs.debian.org (<a href="bugreport.cgi?bug=737778;msg=10">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=10">mbox</a>, <a href="mailto:737778@bugs.debian.org?References=%3C52F308A4.3060408%40redhat.com%3E&amp;subject=Re%3A%20CVE%20request%3A%20f2py%20insecure%20temporary%20file%20use&amp;body=On%20Thu%2C%2006%20Feb%202014%2014%3A59%3A32%20%2B1100%20Murray%20McAllister%20%3Cmmcallis%40redhat.com%3E%20wrote%3A%0A%3E%20Hello%2C%0A%3E%20%0A%3E%20Jakub%20Wilk%20reported%20insecure%20temporary%20file%20use%20in%20f2py.%20From%20%0A%3E%20%3Chttp%3A%2F%2Fbugs.debian.org%2Fcgi-bin%2Fbugreport.cgi%3Fbug%3D737778%3E%3A%0A%3E%20%0A%3E%20%22%22%0A%3E%20numpy%2Ff2py%2F__init__.py%20contains%20this%20code%3A%0A%3E%20%0A%3E%20%20%20%20%20%20%20from%20numpy.distutils.exec_command%20import%20exec_command%0A%3E%20%20%20%20%20%20%20import%20tempfile%0A%3E%20%20%20%20%20%20%20if%20source_fn%20is%20None%3A%0A%3E%20%20%20%20%20%20%20%20%20%20%20fname%20%3D%20os.path.join%28tempfile.mktemp%28%29%2B%27.f%27%29%0A%3E%20%20%20%20%20%20%20else%3A%0A%3E%20%20%20%20%20%20%20%20%20%20%20fname%20%3D%20source_fn%0A%3E%20%0A%3E%20%20%20%20%20%20%20f%20%3D%20open%28fname%2C%27w%27%29%0A%3E%20%22%22%0A%3E%20%0A%3E%20Can%20a%20CVE%20please%20be%20assigned%20if%20one%20hasn%27t%20been%20already%3F%0A%3E%20%0A%3E%20References%3A%0A%3E%20http%3A%2F%2Fbugs.debian.org%2Fcgi-bin%2Fbugreport.cgi%3Fbug%3D737778%0A%3E%20https%3A%2F%2Fbugzilla.redhat.com%2Fshow_bug.cgi%3Fid%3D1062009%0A%3E%20%0A%3E%20Thanks%2C%0A%3E%20%0A%3E%20--%0A%3E%20Murray%20McAllister%20%2F%20Red%20Hat%20Security%20Response%20Team%0A%3E%20%0A%3E%20%0A&amp;In-Reply-To=%3C52F308A4.3060408%40redhat.com%3E">reply</a>):</p>
<div class="headers">
<img src="/cgi-bin/libravatar.cgi?email=mmcallis%40redhat.com" alt="">
<div class="header"><span class="headerfield">From:</span> Murray McAllister &lt;mmcallis@redhat.com&gt;</div>
<div class="header"><span class="headerfield">To:</span> oss-security@lists.openwall.com</div>
<div class="header"><span class="headerfield">Cc:</span> 737778@bugs.debian.org</div>
<div class="header"><span class="headerfield">Subject:</span> CVE request: f2py insecure temporary file use</div>
<div class="header"><span class="headerfield">Date:</span> Thu, 06 Feb 2014 14:59:32 +1100</div>
</div>
<pre class="message flowed">Hello,

Jakub Wilk reported insecure temporary file use in f2py. From 
&lt;<a href="http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=737778">http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=737778</a>&gt;:

&quot;&quot;
numpy/f2py/__init__.py contains this code:

     from numpy.distutils.exec_command import exec_command
     import tempfile
     if source_fn is None:
         fname = os.path.join(tempfile.mktemp()+&#39;.f&#39;)
     else:
         fname = source_fn

     f = open(fname,&#39;w&#39;)
&quot;&quot;

Can a CVE please be assigned if one hasn&#39;t been already?

References:
<a href="http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=737778">http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=737778</a>
<a href="https://bugzilla.redhat.com/show_bug.cgi?id=1062009">https://bugzilla.redhat.com/show_bug.cgi?id=1062009</a>

Thanks,

--
Murray McAllister / Red Hat Security Response Team


</pre>

<div class="msgreceived"><hr><p>
<a name="11"></a>
<!-- command:tag -->
<!-- requester: Julian Taylor &lt;jtaylor.debian@googlemail.com&gt; -->
<!-- request_addr: control@bugs.debian.org -->
<!-- time:1391675044 -->
<!-- new_data:
$new_data = {
              &#39;keywords&#39; =&gt; &#39;security patch&#39;
            };
-->
<!-- old_data:
$old_data = {
              &#39;keywords&#39; =&gt; &#39;security&#39;
            };
-->
<strong>Added tag(s) patch.</strong>
Request was from <code>Julian Taylor &lt;jtaylor.debian@googlemail.com&gt;</code>
to <code>control@bugs.debian.org</code>.
 (Thu, 06 Feb 2014 08:24:04 GMT) (<a href="bugreport.cgi?bug=737778;msg=12">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=12">mbox</a>, <a href="#11">link</a>).</p></p></div>

<div class="infmessage"><hr><p>
<a name="13"></a>
<!-- request_addr: debian-bugs-dist@lists.debian.org, Debian Python Modules Team &lt;python-modules-team@lists.alioth.debian.org&gt; -->
<!-- time:1391727071 -->
<strong>Information forwarded</strong>
to <code>debian-bugs-dist@lists.debian.org, Debian Python Modules Team &lt;python-modules-team@lists.alioth.debian.org&gt;</code>:<br>
<code>Bug#737778</code>; Package <code>python-numpy</code>.
 (Thu, 06 Feb 2014 22:51:11 GMT) (<a href="bugreport.cgi?bug=737778;msg=14">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=14">mbox</a>, <a href="#13">link</a>).</p></p></div>

<div class="infmessage"><hr><p>
<a name="15"></a>
<!-- request_addr: mmcallis@redhat.com -->
<!-- time:1391727071 -->
<strong>Acknowledgement sent</strong>
to <code>mmcallis@redhat.com</code>:<br>
Extra info received and forwarded to list.  Copy sent to <code>Debian Python Modules Team &lt;python-modules-team@lists.alioth.debian.org&gt;</code>.
 (Thu, 06 Feb 2014 22:51:11 GMT) (<a href="bugreport.cgi?bug=737778;msg=16">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=16">mbox</a>, <a href="#15">link</a>).</p></p></div>

<hr><p class="msgreceived"><a name="17"></a><a name="msg17"></a><a href="#17">Message #17</a> received at 737778@bugs.debian.org (<a href="bugreport.cgi?bug=737778;msg=17">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=17">mbox</a>, <a href="mailto:737778@bugs.debian.org?subject=Re%3A%20%5Boss-security%5D%20CVE%20request%3A%20f2py%20insecure%20temporary%20file%20use&amp;References=%3C52F308A4.3060408%40redhat.com%3E%0A%20%3C52F410E9.7090907%40redhat.com%3E&amp;body=On%20Fri%2C%2007%20Feb%202014%2009%3A47%3A05%20%2B1100%20Murray%20McAllister%20%3Cmmcallis%40redhat.com%3E%20wrote%3A%0A%3E%20On%2002%2F06%2F2014%2002%3A59%20PM%2C%20Murray%20McAllister%20wrote%3A%0A%3E%20%3E%20Hello%2C%0A%3E%20%3E%0A%3E%20%3E%20Jakub%20Wilk%20reported%20insecure%20temporary%20file%20use%20in%20f2py.%20From%0A%3E%20%3E%20%3Chttp%3A%2F%2Fbugs.debian.org%2Fcgi-bin%2Fbugreport.cgi%3Fbug%3D737778%3E%3A%0A%3E%20%3E%0A%3E%20%3E%20%22%22%0A%3E%20%3E%20numpy%2Ff2py%2F__init__.py%20contains%20this%20code%3A%0A%3E%20%3E%0A%3E%20%3E%20%20%20%20%20%20%20from%20numpy.distutils.exec_command%20import%20exec_command%0A%3E%20%3E%20%20%20%20%20%20%20import%20tempfile%0A%3E%20%3E%20%20%20%20%20%20%20if%20source_fn%20is%20None%3A%0A%3E%20%3E%20%20%20%20%20%20%20%20%20%20%20fname%20%3D%20os.path.join%28tempfile.mktemp%28%29%2B%27.f%27%29%0A%3E%20%3E%20%20%20%20%20%20%20else%3A%0A%3E%20%3E%20%20%20%20%20%20%20%20%20%20%20fname%20%3D%20source_fn%0A%3E%20%3E%0A%3E%20%3E%20%20%20%20%20%20%20f%20%3D%20open%28fname%2C%27w%27%29%0A%3E%20%3E%20%22%22%0A%3E%20%3E%0A%3E%20%3E%20Can%20a%20CVE%20please%20be%20assigned%20if%20one%20hasn%27t%20been%20already%3F%0A%3E%20%3E%0A%3E%20%3E%20References%3A%0A%3E%20%3E%20http%3A%2F%2Fbugs.debian.org%2Fcgi-bin%2Fbugreport.cgi%3Fbug%3D737778%0A%3E%20%3E%20https%3A%2F%2Fbugzilla.redhat.com%2Fshow_bug.cgi%3Fid%3D1062009%0A%3E%20%3E%0A%3E%20%3E%20Thanks%2C%0A%3E%20%0A%3E%20Thomas%20Spura%20noted%20in%20the%20Red%20Hat%20Bugzilla%20that%20a%20patch%20has%20been%20merged%20%0A%3E%20upstream%3A%0A%3E%20%0A%3E%20https%3A%2F%2Fgithub.com%2Fnumpy%2Fnumpy%2Fpull%2F4262%0A%3E%20%0A%3E%20%0A&amp;In-Reply-To=%3C52F410E9.7090907%40redhat.com%3E">reply</a>):</p>
<div class="headers">
<img src="/cgi-bin/libravatar.cgi?email=mmcallis%40redhat.com" alt="">
<div class="header"><span class="headerfield">From:</span> Murray McAllister &lt;mmcallis@redhat.com&gt;</div>
<div class="header"><span class="headerfield">To:</span> oss-security@lists.openwall.com</div>
<div class="header"><span class="headerfield">Cc:</span> 737778@bugs.debian.org</div>
<div class="header"><span class="headerfield">Subject:</span> Re: [oss-security] CVE request: f2py insecure temporary file use</div>
<div class="header"><span class="headerfield">Date:</span> Fri, 07 Feb 2014 09:47:05 +1100</div>
</div>
<pre class="message flowed">On 02/06/2014 02:59 PM, Murray McAllister wrote:
&gt; Hello,
&gt;
&gt; Jakub Wilk reported insecure temporary file use in f2py. From
&gt; &lt;<a href="http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=737778">http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=737778</a>&gt;:
&gt;
&gt; &quot;&quot;
&gt; numpy/f2py/__init__.py contains this code:
&gt;
&gt;       from numpy.distutils.exec_command import exec_command
&gt;       import tempfile
&gt;       if source_fn is None:
&gt;           fname = os.path.join(tempfile.mktemp()+&#39;.f&#39;)
&gt;       else:
&gt;           fname = source_fn
&gt;
&gt;       f = open(fname,&#39;w&#39;)
&gt; &quot;&quot;
&gt;
&gt; Can a CVE please be assigned if one hasn&#39;t been already?
&gt;
&gt; References:
&gt; <a href="http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=737778">http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=737778</a>
&gt; <a href="https://bugzilla.redhat.com/show_bug.cgi?id=1062009">https://bugzilla.redhat.com/show_bug.cgi?id=1062009</a>
&gt;
&gt; Thanks,

Thomas Spura noted in the Red Hat Bugzilla that a patch has been merged 
upstream:

<a href="https://github.com/numpy/numpy/pull/4262">https://github.com/numpy/numpy/pull/4262</a>


</pre>

<div class="infmessage"><hr><p>
<a name="18"></a>
<!-- request_addr: debian-bugs-dist@lists.debian.org, Debian Python Modules Team &lt;python-modules-team@lists.alioth.debian.org&gt; -->
<!-- time:1391825705 -->
<strong>Information forwarded</strong>
to <code>debian-bugs-dist@lists.debian.org, Debian Python Modules Team &lt;python-modules-team@lists.alioth.debian.org&gt;</code>:<br>
<code>Bug#737778</code>; Package <code>python-numpy</code>.
 (Sat, 08 Feb 2014 02:15:05 GMT) (<a href="bugreport.cgi?bug=737778;msg=19">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=19">mbox</a>, <a href="#18">link</a>).</p></p></div>

<div class="infmessage"><hr><p>
<a name="20"></a>
<!-- request_addr: cve-assign@mitre.org -->
<!-- time:1391825705 -->
<strong>Acknowledgement sent</strong>
to <code>cve-assign@mitre.org</code>:<br>
Extra info received and forwarded to list.  Copy sent to <code>Debian Python Modules Team &lt;python-modules-team@lists.alioth.debian.org&gt;</code>.
 (Sat, 08 Feb 2014 02:15:05 GMT) (<a href="bugreport.cgi?bug=737778;msg=21">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=21">mbox</a>, <a href="#20">link</a>).</p></p></div>

<hr><p class="msgreceived"><a name="22"></a><a name="msg22"></a><a href="#22">Message #22</a> received at 737778@bugs.debian.org (<a href="bugreport.cgi?bug=737778;msg=22">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=22">mbox</a>, <a href="mailto:737778@bugs.debian.org?subject=Re%3A%20CVE%20request%3A%20f2py%20insecure%20temporary%20file%20use&amp;References=%3C201402080206.s18260Gd005187%40linus.mitre.org%3E&amp;body=On%20Fri%2C%207%20Feb%202014%2021%3A06%3A00%20-0500%20%28EST%29%20cve-assign%40mitre.org%20wrote%3A%0A%3E%20-----BEGIN%20PGP%20SIGNED%20MESSAGE-----%0A%3E%20Hash%3A%20SHA1%0A%3E%20%0A%3E%20%3E%20Jakub%20Wilk%20reported%20insecure%20temporary%20file%20use%20in%20f2py.%0A%3E%20%3E%20%0A%3E%20%3E%20numpy%2Ff2py%2F__init__.py%20contains%20this%20code%3A%0A%3E%20%3E%20%0A%3E%20%3E%20%20%20%20%20%20%20%20%20%20%20fname%20%3D%20os.path.join%28tempfile.mktemp%28%29%2B%27.f%27%29%0A%3E%20%3E%20%0A%3E%20%3E%20%20%20%20%20%20%20f%20%3D%20open%28fname%2C%27w%27%29%0A%3E%20%3E%20%0A%3E%20%3E%20Can%20a%20CVE%20please%20be%20assigned%20if%20one%20hasn%27t%20been%20already%3F%0A%3E%20%3E%20%0A%3E%20%3E%20http%3A%2F%2Fbugs.debian.org%2Fcgi-bin%2Fbugreport.cgi%3Fbug%3D737778%0A%3E%20%3E%20https%3A%2F%2Fbugzilla.redhat.com%2Fshow_bug.cgi%3Fid%3D1062009%0A%3E%20%0A%3E%20Use%20CVE-2014-1858%20only%20for%20the%20issue%20in%20the%20__init__.py%20file.%0A%3E%20%0A%3E%20Use%20CVE-2014-1859%20for%20the%20other%20temporary-file%20issues%20fixed%20by%20the%0A%3E%20vendor%20in%20the%0A%3E%20https%3A%2F%2Fgithub.com%2Fnumpy%2Fnumpy%2Fcommit%2F0bb46c1448b0d3f5453d5182a17ea7ac5854ee15%0A%3E%20commit.%0A%3E%20%0A%3E%20-%20--%20%0A%3E%20CVE%20assignment%20team%2C%20MITRE%20CVE%20Numbering%20Authority%0A%3E%20M%2FS%20M300%0A%3E%20202%20Burlington%20Road%2C%20Bedford%2C%20MA%2001730%20USA%0A%3E%20%5B%20PGP%20key%20available%20through%20http%3A%2F%2Fcve.mitre.org%2Fcve%2Frequest_id.html%20%5D%0A%3E%20-----BEGIN%20PGP%20SIGNATURE-----%0A%3E%20Version%3A%20GnuPG%20v1.4.14%20%28SunOS%29%0A%3E%20%0A%3E%20iQEcBAEBAgAGBQJS9Y9iAAoJEKllVAevmvmsmUgH%2FjW37Wa7Wp52niRfZ%2B5B3IR%2B%0A%3E%20emZwCRGRhJKZVZKB3yWDPOLv7WPGsXMQUgRzNLI81U2ukGX5%2BZDQCAvm2o5fed25%0A%3E%20z90k82ER5lwmbosp87p%2FkKNtCTuLegijDczduBIV73fO3PwC1d%2B%2FJM5I4%2FDnTSM6%0A%3E%20OWLRquY7giwDPiF5NvBrmDR6JocWOPVlbAHoIvLuxRFcYdFbqDaJe8Bt8hf2saQB%0A%3E%20Phw%2FnIaladkNJOKR5sZM9%2BE3tVdP1MPCjmiMdASWktTP0fNrGMoBS24zTAQY5hgT%0A%3E%20ApAW%2B6Y88igBbZ%2Faci5kvIo7ocdmw%2Bld7YNK46PMX8Cr4MsTJZX0X6V85HCzAJM%3D%0A%3E%20%3DXwId%0A%3E%20-----END%20PGP%20SIGNATURE-----%0A%3E%20%0A%3E%20%0A&amp;In-Reply-To=%3C201402080206.s18260Gd005187%40linus.mitre.org%3E">reply</a>):</p>
<div class="headers">
<img src="/cgi-bin/libravatar.cgi?email=cve-assign%40mitre.org" alt="">
<div class="header"><span class="headerfield">From:</span> cve-assign@mitre.org</div>
<div class="header"><span class="headerfield">To:</span> mmcallis@redhat.com</div>
<div class="header"><span class="headerfield">Cc:</span> cve-assign@mitre.org, oss-security@lists.openwall.com,
        737778@bugs.debian.org</div>
<div class="header"><span class="headerfield">Subject:</span> Re: CVE request: f2py insecure temporary file use</div>
<div class="header"><span class="headerfield">Date:</span> Fri, 7 Feb 2014 21:06:00 -0500 (EST)</div>
</div>
<pre class="message">-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

&gt; Jakub Wilk reported insecure temporary file use in f2py.
&gt; 
&gt; numpy/f2py/__init__.py contains this code:
&gt; 
&gt;           fname = os.path.join(tempfile.mktemp()+&#39;.f&#39;)
&gt; 
&gt;       f = open(fname,&#39;w&#39;)
&gt; 
&gt; Can a CVE please be assigned if one hasn&#39;t been already?
&gt; 
&gt; <a href="http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=737778">http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=737778</a>
&gt; <a href="https://bugzilla.redhat.com/show_bug.cgi?id=1062009">https://bugzilla.redhat.com/show_bug.cgi?id=1062009</a>

Use <a href="https://security-tracker.debian.org/tracker/CVE-2014-1858">CVE-2014-1858</a> only for the issue in the __init__.py file.

Use <a href="https://security-tracker.debian.org/tracker/CVE-2014-1859">CVE-2014-1859</a> for the other temporary-file issues fixed by the
vendor in the
<a href="https://github.com/numpy/numpy/commit/0bb46c1448b0d3f5453d5182a17ea7ac5854ee15">https://github.com/numpy/numpy/commit/0bb46c1448b0d3f5453d5182a17ea7ac5854ee15</a>
commit.

- -- 
CVE assignment team, MITRE CVE Numbering Authority
M/S M300
202 Burlington Road, Bedford, MA 01730 USA
[ PGP key available through <a href="http://cve.mitre.org/cve/request_id.html">http://cve.mitre.org/cve/request_id.html</a> ]
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v1.4.14 (SunOS)

iQEcBAEBAgAGBQJS9Y9iAAoJEKllVAevmvmsmUgH/jW37Wa7Wp52niRfZ+5B3IR+
emZwCRGRhJKZVZKB3yWDPOLv7WPGsXMQUgRzNLI81U2ukGX5+ZDQCAvm2o5fed25
z90k82ER5lwmbosp87p/kKNtCTuLegijDczduBIV73fO3PwC1d+/JM5I4/DnTSM6
OWLRquY7giwDPiF5NvBrmDR6JocWOPVlbAHoIvLuxRFcYdFbqDaJe8Bt8hf2saQB
Phw/nIaladkNJOKR5sZM9+E3tVdP1MPCjmiMdASWktTP0fNrGMoBS24zTAQY5hgT
ApAW+6Y88igBbZ/aci5kvIo7ocdmw+ld7YNK46PMX8Cr4MsTJZX0X6V85HCzAJM=
=XwId
-----END PGP SIGNATURE-----


</pre>

<div class="msgreceived"><hr><p>
<a name="23"></a>
<!-- command:title -->
<!-- requester: Salvatore Bonaccorso &lt;carnil@debian.org&gt; -->
<!-- request_addr: control@bugs.debian.org -->
<!-- time:1391844978 -->
<!-- new_data:
$new_data = {
              &#39;subject&#39; =&gt; &#39;python-numpy: insecure use of /tmp (CVE-2014-1858 CVE-2014-1859)&#39;
            };
-->
<!-- old_data:
$old_data = {
              &#39;subject&#39; =&gt; &#39;python-numpy: insecure use of /tmp&#39;
            };
-->
<strong>Changed Bug title to &#39;python-numpy: insecure use of /tmp (CVE-2014-1858 CVE-2014-1859)&#39; from &#39;python-numpy: insecure use of /tmp&#39;</strong>
Request was from <code>Salvatore Bonaccorso &lt;carnil@debian.org&gt;</code>
to <code>control@bugs.debian.org</code>.
 (Sat, 08 Feb 2014 07:36:18 GMT) (<a href="bugreport.cgi?bug=737778;msg=24">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=24">mbox</a>, <a href="#23">link</a>).</p></p></div>

<div class="msgreceived"><hr><p>
<a name="25"></a>
<!-- command:tag -->
<!-- requester: jtaylor-guest@users.alioth.debian.org -->
<!-- request_addr: control@bugs.debian.org -->
<!-- time:1393775648 -->
<!-- new_data:
$new_data = {
              &#39;keywords&#39; =&gt; &#39;security pending patch&#39;
            };
-->
<!-- old_data:
$old_data = {
              &#39;keywords&#39; =&gt; &#39;security patch&#39;
            };
-->
<strong>Added tag(s) pending.</strong>
Request was from <code>jtaylor-guest@users.alioth.debian.org</code>
to <code>control@bugs.debian.org</code>.
 (Sun, 02 Mar 2014 15:54:08 GMT) (<a href="bugreport.cgi?bug=737778;msg=26">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=26">mbox</a>, <a href="#25">link</a>).</p></p></div>

<div class="msgreceived"><hr><p>
<a name="27"></a>
<!-- request_addr: Julian Taylor &lt;jtaylor.debian@googlemail.com&gt; -->
<!-- time:1394659457 -->
<strong>Reply sent</strong>
to <code>Julian Taylor &lt;jtaylor.debian@googlemail.com&gt;</code>:<br>
You have taken responsibility.
 (Wed, 12 Mar 2014 21:24:17 GMT) (<a href="bugreport.cgi?bug=737778;msg=28">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=28">mbox</a>, <a href="#27">link</a>).</p></p></div>

<div class="infmessage"><hr><p>
<a name="29"></a>
<!-- request_addr: Jakub Wilk &lt;jwilk@debian.org&gt; -->
<!-- time:1394659457 -->
<strong>Notification sent</strong>
to <code>Jakub Wilk &lt;jwilk@debian.org&gt;</code>:<br>
Bug acknowledged by developer.
 (Wed, 12 Mar 2014 21:24:17 GMT) (<a href="bugreport.cgi?bug=737778;msg=30">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=30">mbox</a>, <a href="#29">link</a>).</p></p></div>

<hr><p class="msgreceived"><a name="31"></a><a name="msg31"></a><a href="#31">Message #31</a> received at 737778-close@bugs.debian.org (<a href="bugreport.cgi?bug=737778;msg=31">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=31">mbox</a>, <a href="mailto:737778@bugs.debian.org?References=%3CE1WNqao-0000zM-CJ%40franck.debian.org%3E&amp;subject=Re%3A%20Bug%23737778%3A%20fixed%20in%20python-numpy%201%3A1.8.1~rc1-1&amp;In-Reply-To=%3CE1WNqao-0000zM-CJ%40franck.debian.org%3E&amp;body=On%20Wed%2C%2012%20Mar%202014%2021%3A21%3A38%20%2B0000%20Julian%20Taylor%20%3Cjtaylor.debian%40googlemail.com%3E%20wrote%3A%0A%3E%20Source%3A%20python-numpy%0A%3E%20Source-Version%3A%201%3A1.8.1~rc1-1%0A%3E%20%0A%3E%20We%20believe%20that%20the%20bug%20you%20reported%20is%20fixed%20in%20the%20latest%20version%20of%0A%3E%20python-numpy%2C%20which%20is%20due%20to%20be%20installed%20in%20the%20Debian%20FTP%20archive.%0A%3E%20%0A%3E%20A%20summary%20of%20the%20changes%20between%20this%20version%20and%20the%20previous%20one%20is%0A%3E%20attached.%0A%3E%20%0A%3E%20Thank%20you%20for%20reporting%20the%20bug%2C%20which%20will%20now%20be%20closed.%20%20If%20you%0A%3E%20have%20further%20comments%20please%20address%20them%20to%20737778%40bugs.debian.org%2C%0A%3E%20and%20the%20maintainer%20will%20reopen%20the%20bug%20report%20if%20appropriate.%0A%3E%20%0A%3E%20Debian%20distribution%20maintenance%20software%0A%3E%20pp.%0A%3E%20Julian%20Taylor%20%3Cjtaylor.debian%40googlemail.com%3E%20%28supplier%20of%20updated%20python-numpy%20package%29%0A%3E%20%0A%3E%20%28This%20message%20was%20generated%20automatically%20at%20their%20request%3B%20if%20you%0A%3E%20believe%20that%20there%20is%20a%20problem%20with%20it%20please%20contact%20the%20archive%0A%3E%20administrators%20by%20mailing%20ftpmaster%40ftp-master.debian.org%29%0A%3E%20%0A%3E%20%0A%3E%20-----BEGIN%20PGP%20SIGNED%20MESSAGE-----%0A%3E%20Hash%3A%20SHA1%0A%3E%20%0A%3E%20Format%3A%201.8%0A%3E%20Date%3A%20Sun%2C%2002%20Mar%202014%2015%3A33%3A25%20%2B0100%0A%3E%20Source%3A%20python-numpy%0A%3E%20Binary%3A%20python-numpy%20python-numpy-dbg%20python3-numpy%20python3-numpy-dbg%20python-numpy-doc%0A%3E%20Architecture%3A%20source%20amd64%20all%0A%3E%20Version%3A%201%3A1.8.1~rc1-1%0A%3E%20Distribution%3A%20unstable%0A%3E%20Urgency%3A%20low%0A%3E%20Maintainer%3A%20Debian%20Python%20Modules%20Team%20%3Cpython-modules-team%40lists.alioth.debian.org%3E%0A%3E%20Changed-By%3A%20Julian%20Taylor%20%3Cjtaylor.debian%40googlemail.com%3E%0A%3E%20Description%3A%20%0A%3E%20%20python-numpy%20-%20Numerical%20Python%20adds%20a%20fast%20array%20facility%20to%20the%20Python%20languag%0A%3E%20%20python-numpy-dbg%20-%20Fast%20array%20facility%20to%20the%20Python%20language%20%28debug%20extension%29%0A%3E%20%20python-numpy-doc%20-%20NumPy%20documentation%0A%3E%20%20python3-numpy%20-%20Fast%20array%20facility%20to%20the%20Python%203%20language%0A%3E%20%20python3-numpy-dbg%20-%20Fast%20array%20facility%20to%20the%20Python%203%20language%20%28debug%20extension%29%0A%3E%20Closes%3A%20695881%20710177%20724611%20737778%20739019%20740318%0A%3E%20Changes%3A%20%0A%3E%20%20python-numpy%20%281%3A1.8.1~rc1-1%29%20unstable%3B%20urgency%3Dlow%0A%3E%20%20.%0A%3E%20%20%20%20%2A%20New%20upstream%20bugfix%20release%20candidate%0A%3E%20%20%20%20%20%20-%20removed%20python-numpydoc%20from%20b-d%2C%20upstream%20tarballs%20includes%20it%20again%0A%3E%20%20%20%20%20%20-%20fixes%20insecure%20mktemp%20usage%20of%20f2py%20%28Closes%3A%20%23737778%29%0A%3E%20%20%20%20%2A%20add%20autopkgtests%20running%20testsuite%20with%20different%20BLAS%20and%20testing%20f2py%0A%3E%20%20%20%20%20%20and%20distutils%20%28Closes%3A%20%23695881%29%0A%3E%20%20%20%20%2A%20use%20dh_python2%20instead%20of%20deprecated%20pysupport%0A%3E%20%20%20%20%2A%2050_search-multiarch-paths.patch%3A%20drop%2C%20applied%20upstream%0A%3E%20%20%20%20%2A%20build%20depend%20on%20cython%20and%20cythonize%20mtrand.pyx%20%28Closes%3A%20%23710177%29%0A%3E%20%20%20%20%2A%20move%20documentation%20build%20depends%20to%20-indep%20%28Closes%3A%20%23739019%29%0A%3E%20%20%20%20%2A%20run%20tests%20in%20verbose%20mode%20%28Closes%3A%20%23724611%29%0A%3E%20%20%20%20%2A%20python3-soabi.patch%3A%20fix%20ctypeslib%20for%20python3%20soabi%20in%20extension%20filenames%0A%3E%20%20%20%20%2A%20debian%2Fpython3-numpy-dbg.install%3A%0A%3E%20%20%20%20%20%20-%20fix%20duplicate%20files%20in%20dbg%20package%20of%20kfreebsd%20%28Closes%3A%20%23740318%29%0A%3E%20%20%20%20%2A%20bump%20Standards-Version%20to%203.9.5%20%28no%20changes%20needed%29%0A%3E%20%20%20%20%2A%20restore-3kcompat-api.patch%3A%0A">reply</a>):</p>
<div class="headers">
<img src="/cgi-bin/libravatar.cgi?email=jtaylor.debian%40googlemail.com" alt="">
<div class="header"><span class="headerfield">From:</span> Julian Taylor &lt;jtaylor.debian@googlemail.com&gt;</div>
<div class="header"><span class="headerfield">To:</span> 737778-close@bugs.debian.org</div>
<div class="header"><span class="headerfield">Subject:</span> Bug#737778: fixed in python-numpy 1:1.8.1~rc1-1</div>
<div class="header"><span class="headerfield">Date:</span> Wed, 12 Mar 2014 21:21:38 +0000</div>
</div>
<pre class="message">Source: python-numpy
Source-Version: 1:1.8.1~rc1-1

We believe that the bug you reported is fixed in the latest version of
python-numpy, which is due to be installed in the Debian FTP archive.

A summary of the changes between this version and the previous one is
attached.

Thank you for reporting the bug, which will now be closed.  If you
have further comments please address them to 737778@bugs.debian.org,
and the maintainer will reopen the bug report if appropriate.

Debian distribution maintenance software
pp.
Julian Taylor &lt;jtaylor.debian@googlemail.com&gt; (supplier of updated python-numpy package)

(This message was generated automatically at their request; if you
believe that there is a problem with it please contact the archive
administrators by mailing ftpmaster@ftp-master.debian.org)


-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

Format: 1.8
Date: Sun, 02 Mar 2014 15:33:25 +0100
Source: python-numpy
Binary: python-numpy python-numpy-dbg python3-numpy python3-numpy-dbg python-numpy-doc
Architecture: source amd64 all
Version: 1:1.8.1~rc1-1
Distribution: unstable
Urgency: low
Maintainer: Debian Python Modules Team &lt;python-modules-team@lists.alioth.debian.org&gt;
Changed-By: Julian Taylor &lt;jtaylor.debian@googlemail.com&gt;
Description: 
 python-numpy - Numerical Python adds a fast array facility to the Python languag
 python-numpy-dbg - Fast array facility to the Python language (debug extension)
 python-numpy-doc - NumPy documentation
 python3-numpy - Fast array facility to the Python 3 language
 python3-numpy-dbg - Fast array facility to the Python 3 language (debug extension)
Closes: <a href="bugreport.cgi?bug=695881">695881</a> <a href="bugreport.cgi?bug=710177">710177</a> <a href="bugreport.cgi?bug=724611">724611</a> <a href="bugreport.cgi?bug=737778">737778</a> <a href="bugreport.cgi?bug=739019">739019</a> <a href="bugreport.cgi?bug=740318">740318</a>
Changes: 
 python-numpy (1:1.8.1~rc1-1) unstable; urgency=low
 .
   * New upstream bugfix release candidate
     - removed python-numpydoc from b-d, upstream tarballs includes it again
     - fixes insecure mktemp usage of f2py (Closes: #<a href="bugreport.cgi?bug=737778">737778</a>)
   * add autopkgtests running testsuite with different BLAS and testing f2py
     and distutils (Closes: #<a href="bugreport.cgi?bug=695881">695881</a>)
   * use dh_python2 instead of deprecated pysupport
   * 50_search-multiarch-paths.patch: drop, applied upstream
   * build depend on cython and cythonize mtrand.pyx (Closes: #<a href="bugreport.cgi?bug=710177">710177</a>)
   * move documentation build depends to -indep (Closes: #<a href="bugreport.cgi?bug=739019">739019</a>)
   * run tests in verbose mode (Closes: #<a href="bugreport.cgi?bug=724611">724611</a>)
   * python3-soabi.patch: fix ctypeslib for python3 soabi in extension filenames
   * debian/python3-numpy-dbg.install:
     - fix duplicate files in dbg package of kfreebsd (Closes: #<a href="bugreport.cgi?bug=740318">740318</a>)
   * bump Standards-Version to 3.9.5 (no changes needed)
   * restore-3kcompat-api.patch:
     add upstream patch to restore private api used by matplotlib
Checksums-Sha1: 
 5eb1d7b49d487bf60ed77b59455c463cdf616426 1938 python-numpy_1.8.1~rc1-1.dsc
 873ab03d189e5a0687ba79d55033a7f30be20d9d 3790496 python-numpy_1.8.1~rc1.orig.tar.gz
 39fbc1b4f6b8908a57d7d4ec0a6408eb6ebe7cd7 140204 python-numpy_1.8.1~rc1-1.debian.tar.xz
 5e4311d7f0f797573ff7a55f7796a704fda73ac8 1600168 python-numpy_1.8.1~rc1-1_amd64.deb
 0c10f6d67ac3c32abbc23537b817f9f076e97ae8 2969766 python-numpy-dbg_1.8.1~rc1-1_amd64.deb
 6f7639cdb01af9e5306c4a67c201ecc777107ab5 1668996 python3-numpy_1.8.1~rc1-1_amd64.deb
 f40a022bf9f2d9fce96702c56f59740c30ab0bbe 5363806 python3-numpy-dbg_1.8.1~rc1-1_amd64.deb
 8e95298a11a7f89c22ff73dee6f2e5544491e371 3928026 python-numpy-doc_1.8.1~rc1-1_all.deb
Checksums-Sha256: 
 d43cffd1340570b757d355dceb0b730b4a13e7f78abd90083cb47a8a6fe3f4ec 1938 python-numpy_1.8.1~rc1-1.dsc
 78259bf9d770f79665a1622325094c356bff739cb4ce5236b49e707855404ba4 3790496 python-numpy_1.8.1~rc1.orig.tar.gz
 ca66f3875c592a2eb32a1fa8c3faed00b17005ae7f8c44d34ec6303d7bb4e517 140204 python-numpy_1.8.1~rc1-1.debian.tar.xz
 c2efd5bd0ef1dc9113ace09fee4a6433b6171ae070e3ad1f5d3da50c80b09a86 1600168 python-numpy_1.8.1~rc1-1_amd64.deb
 963b261964754bb82293dfb60f7d81c2e9ed03b37bb95fa96c85f742da19cb58 2969766 python-numpy-dbg_1.8.1~rc1-1_amd64.deb
 ad95e8f6fd3301b187c7084fcee88dfc614fff2142f364f5c6720145c6ccfba5 1668996 python3-numpy_1.8.1~rc1-1_amd64.deb
 0741fbb32ead688ab706ed396fcddefb18a33754acf231292178038fb7a25afa 5363806 python3-numpy-dbg_1.8.1~rc1-1_amd64.deb
 367c7b74cd50e87dc10916e7b40fa42382b5f5d914881da2844f7733fd354d62 3928026 python-numpy-doc_1.8.1~rc1-1_all.deb
Files: 
 288c0cd5f73f75d63e62fc59415103e9 1938 python optional python-numpy_1.8.1~rc1-1.dsc
 a9a48950271fc2eac56a5d2418783279 3790496 python optional python-numpy_1.8.1~rc1.orig.tar.gz
 d9040651b5bc477de29c5954d51a9b85 140204 python optional python-numpy_1.8.1~rc1-1.debian.tar.xz
 831d385e339e332a63422da64d9423fb 1600168 python optional python-numpy_1.8.1~rc1-1_amd64.deb
 bcc60a034b4be309e1cc0da6d8b388ee 2969766 debug extra python-numpy-dbg_1.8.1~rc1-1_amd64.deb
 3d8c5d7d7ce56b31b869cfdae483139f 1668996 python optional python3-numpy_1.8.1~rc1-1_amd64.deb
 66895eaa991544e6e0d577632845acae 5363806 debug extra python3-numpy-dbg_1.8.1~rc1-1_amd64.deb
 0274656a722e5526c8930f09d5a1bb28 3928026 doc optional python-numpy-doc_1.8.1~rc1-1_all.deb

-----BEGIN PGP SIGNATURE-----
Version: GnuPG v1.4.14 (GNU/Linux)

iEYEARECAAYFAlMguXAACgkQAukwV0RN2VAH7ACfYdLLMpU6YnhcwE3MJLwMB08X
xBgAmwWSkbFxz5PMQGhOQLJrvZmwRojo
=+Xad
-----END PGP SIGNATURE-----



</pre>

<div class="msgreceived"><hr><p>
<a name="32"></a>
<!-- command:archive -->
<!-- requester: Debbugs Internal Request &lt;owner@bugs.debian.org&gt; -->
<!-- request_addr: internal_control@bugs.debian.org -->
<!-- time:1397115038 -->
<!-- new_data:
$new_data = {};
-->
<!-- old_data:
$old_data = {};
-->
<strong>Bug archived.</strong>
Request was from <code>Debbugs Internal Request &lt;owner@bugs.debian.org&gt;</code>
to <code>internal_control@bugs.debian.org</code>.
 (Thu, 10 Apr 2014 07:30:38 GMT) (<a href="bugreport.cgi?bug=737778;msg=33">full text</a>, <a href="bugreport.cgi?bug=737778;mbox=yes;msg=33">mbox</a>, <a href="#32">link</a>).</p></p></div>

<hr>
<p class="msgreceived">Send a report that <a href="https://bugs.debian.org/cgi-bin/bugspam.cgi?bug=737778">this bug log contains spam</a>.</p>
<hr>
<ADDRESS>Debian bug tracking system administrator &lt;<A HREF="mailto:owner@bugs.debian.org">owner@bugs.debian.org</A>&gt;.
Last modified:
<!--timestamp-->Sat Oct 30 18:28:45 2021<!--end timestamp-->; 
Machine Name:
<!--machinename-->buxtehude<!--machinename-->
<P>
<A HREF="https://www.debian.org/Bugs/">Debian Bug tracking system</A>
</p>
<p>
  Debbugs is free software and licensed under the terms of the GNU
  Public License version 2. The current version can be obtained
  from <a href="https://bugs.debian.org/debbugs-source/">https://bugs.debian.org/debbugs-source/</a>.
</p>
<p>
Copyright © 1999 Darren O. Benham,
1997,2003 nCipher Corporation Ltd,
1994-97 Ian Jackson,
2005-2017 Don Armstrong, and many other contributors.
</p>
</ADDRESS>

</body>
</html>
